src/main/java/io/github/thebusybiscuit/slimefun4/core/attributes/RandomMobDrop.java

```diff
@@ -26,7 +27,7 @@
     /**
      * Implement this method to make the object have a variable chance of being
      * added to the dropList when {@link EntityType} (specified in the recipe)
-     * is killed by the {@link Player}
+     * is killed by the {@link Player} {@link RecipeType#MOB_DROP} 

```

Comment:
This doesn't make sense anymore. "killed by the Player RecipeType#MOB_DROP".

Should probably be mentioned above when the entity and recipe is mentioned.

Reply:
fixed